.github/workflows/compile_protos.yml

```diff
@@ -64,16 +54,19 @@ jobs:
         if: startsWith(env.commit-message, env.CI_COMMIT_MESSAGE_PREFIX) && env.commit-author == env.CI_COMMIT_AUTHOR
         run: echo "is-auto-commit=true" >> $GITHUB_ENV
 
+      - uses: jdx/mise-action@v2
+        if: ${{ !env.is-auto-commit }}
+        with:
+          experimental: true
+
       - uses: bufbuild/buf-setup-action@v1
         if: ${{ !env.is-auto-commit }}
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-      - uses: arduino/setup-protoc@v1
-        if: ${{ !env.is-auto-commit }}
 
       - name: Compile protos
         if: ${{ !env.is-auto-commit }}
-        run: make dist/buf
+        run: 'mise r --force clean ::: buf'
```

Comment:
expand `r` to `run` imo -- makes it more accessible for whoever touches this next